src/components/Receiving/ItemDetails.js

```diff
@@ -31,61 +31,85 @@ class ItemDetails extends Component {
     super(props);
 
     this.state = {
-      allChecked: false,
+      allChecked: {},
       checkedItems: [],
+      currentLine: 0,
+      lineItems: this.props.linesItemList,
```

Comment:
I don't see any reason to do that. Do you?

Reply:
To update barcode, locationId.